code/__DEFINES/misc.dm

```diff
@@ -384,7 +384,7 @@
 #define EXPLOSIVE_WALL_GROUP_SYNDICATE_BASE "syndicate_base"
 
 // Filters
-#define FILTER_AMBIENT_OCCLUSION filter(type="drop_shadow", x=0, y=-2, size=4, border=4, color="#04080FAA")
+#define FILTER_AMBIENT_OCCLUSION filter(type="drop_shadow", x=0, y=-2, size=4, color="#04080FAA")
```

Comment:
What impact does the loss of the `border` parameter have?

Reply:
I've not noticed its impact yet, it may need more testing but it doesn't seem large enough to be noticeable